 Makefile     |  2 +-
 config.def.h | 63 +++++++++++++++++++++++++++-------------------------
 2 files changed, 34 insertions(+), 31 deletions(-)

diff --git a/Makefile b/Makefile
index 470ac86..4d49649 100644
--- a/Makefile
+++ b/Makefile
@@ -30,7 +30,7 @@ st: $(OBJ)
 	$(CC) -o $@ $(OBJ) $(STLDFLAGS)
 
 clean:
-	rm -f st $(OBJ) st-$(VERSION).tar.gz
+	rm -f st $(OBJ) st-$(VERSION).tar.gz config.h
 
 dist: clean
 	mkdir -p st-$(VERSION)
diff --git a/config.def.h b/config.def.h
index e3b469b..e8d1a95 100644
--- a/config.def.h
+++ b/config.def.h
@@ -5,7 +5,7 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-static char *font = "Liberation Mono:pixelsize=12:antialias=true:autohint=true";
+static char *font = "Liberation Mono:pixelsize=15:antialias=true:autohint=true";
 static int borderpx = 2;
 
 /*
@@ -95,33 +95,34 @@ unsigned int tabspaces = 8;
 
 /* Terminal colors (16 first used in escape sequence) */
 static const char *colorname[] = {
-	/* 8 normal colors */
-	"black",
-	"red3",
-	"green3",
-	"yellow3",
-	"blue2",
-	"magenta3",
-	"cyan3",
-	"gray90",
-
-	/* 8 bright colors */
-	"gray50",
-	"red",
-	"green",
-	"yellow",
-	"#5c5cff",
-	"magenta",
-	"cyan",
-	"white",
-
-	[255] = 0,
-
-	/* more colors can be added after 255 to use with DefaultXX */
-	"#cccccc",
-	"#555555",
-	"gray90", /* default foreground colour */
-	"black", /* default background colour */
+    /* 8 normal colors */
+    "#06080a",
+    "#ee6d85",
+    "#95c561",
+    "#d7a65f",
+    "#7199ee",
+    "#a485dd",
+    "#38a89d",
+    "#a0a8cd",
+
+    /* 8 bright colors */
+    "#212234",
+    "#ee6d85",
+    "#95c561",
+    "#d7a65f",
+    "#7199ee",
+    "#a485dd",
+    "#38a89d",
+    "#a0a8cd",
+
+    [255] = 0,
+
+    /* more colors can be added after 255 to use with DefaultXX */
+
+    "#a0a8cd",
+    "#33467c",
+    "#a0a8cd", /* default foreground colour */
+    "#11121d", /* default background colour */
 };
 
 
@@ -201,8 +202,10 @@ static Shortcut shortcuts[] = {
 	{ TERMMOD,              XK_Y,           selpaste,       {.i =  0} },
 	{ ShiftMask,            XK_Insert,      selpaste,       {.i =  0} },
 	{ TERMMOD,              XK_Num_Lock,    numlock,        {.i =  0} },
-	{ ShiftMask,            XK_Page_Up,     kscrollup,      {.i = -1} },
-	{ ShiftMask,            XK_Page_Down,   kscrolldown,    {.i = -1} },
+	{ ControlMask,          XK_k,           kscrollup,      {.i =  3} },
+	{ ControlMask,          XK_j,           kscrolldown,    {.i =  3} },
+	{ TERMMOD,              XK_K,           kscrollup,      {.i = 30} },
+	{ TERMMOD,              XK_J,           kscrolldown,    {.i = 30} },
 };
 
 /*
-- 
2.42.0

