 Makefile             |  2 +-
 components/battery.c |  4 ++--
 config.def.h         | 17 ++++++++++++++---
 3 files changed, 17 insertions(+), 6 deletions(-)

diff --git a/Makefile b/Makefile
index 7a18274..b48578c 100644
--- a/Makefile
+++ b/Makefile
@@ -44,7 +44,7 @@ slstatus: slstatus.o $(COM:=.o) $(REQ:=.o)
 	$(CC) -o $@ $(LDFLAGS) $(COM:=.o) $(REQ:=.o) slstatus.o $(LDLIBS)
 
 clean:
-	rm -f slstatus slstatus.o $(COM:=.o) $(REQ:=.o) slstatus-${VERSION}.tar.gz
+	rm -f slstatus slstatus.o $(COM:=.o) $(REQ:=.o) slstatus-${VERSION}.tar.gz config.h
 
 dist:
 	rm -rf "slstatus-$(VERSION)"
diff --git a/components/battery.c b/components/battery.c
index 1c753f9..2318d66 100644
--- a/components/battery.c
+++ b/components/battery.c
@@ -58,8 +58,8 @@
 		} map[] = {
 			{ "Charging",    "+" },
 			{ "Discharging", "-" },
-			{ "Full",        "o" },
-			{ "Not charging", "o" },
+			{ "Full",        "" },
+			{ "Not charging", "?" },
 		};
 		size_t i;
 		char path[PATH_MAX], state[12];
diff --git a/config.def.h b/config.def.h
index d805331..910d11d 100644
--- a/config.def.h
+++ b/config.def.h
@@ -1,10 +1,12 @@
 /* See LICENSE file for copyright and license details. */
 
+#define LAPTOP 1
+
 /* interval between updates (in ms) */
 const unsigned int interval = 1000;
 
 /* text to show if no value can be retrieved */
-static const char unknown_str[] = "n/a";
+static const char unknown_str[] = "(?)";
 
 /* maximum output string length */
 #define MAXLEN 2048
@@ -64,6 +66,15 @@ static const char unknown_str[] = "n/a";
  * wifi_perc           WiFi signal in percent          interface name (wlan0)
  */
 static const struct arg args[] = {
-	/* function format          argument */
-	{ datetime, "%s",           "%F %T" },
+	/* function       format           argument */
+	{  netspeed_rx,   "Down %sB/s | ", "wlp4s0" },
+	{  netspeed_tx,   "Up %sB/s | ",   "wlp4s0" },
+	{  cpu_perc,      "CPU %s%% | ",   NULL },
+	{  ram_perc,      "RAM %s%% | ",   NULL },
+	{  run_command,   "Vol %s | ",     "amixer get Master | tail -1 | awk -F \"[][]\" '/%/ { print $2 \" \" $4 }'" },
+#if LAPTOP
+	{  battery_perc,  "Bat %s%%",      "BAT0" },
+	{  battery_state, "%s | ",         "BAT0" },
+#endif
+	{  datetime,      "%s",             "%a %d/%m %T" },
 };
-- 
2.41.0

